dashpay/src/main/kotlin/org/dashevo/dashpay/ContactRequests.kt

```diff
@@ -57,4 +89,30 @@ class ContactRequests(val platform: Platform) {
 
         return documents
     }
+
+    suspend fun watchContactRequest(
+        fromUserId: String,
+        toUserId: String,
+        retryCount: Int,
+        delayMillis: Long,
+        retryDelayType: RetryDelayType
+    ): Document? {
+        val documentQuery = DocumentQuery.Builder()
+        documentQuery.where(listOf("\$userId" to fromUserId, "toUserId" to toUserId))
```

Comment:
This results in an invalid query.

```kotlin
documentQuery.where(listOf("\$userId", "==", fromUserId))
  .where("toUserId", "==", toUserId))

```

A where clause in a query must contain three items.  First is the field, second is the operator and lastly is the value.  In this case, there should be two where clauses.